src/components/AsideBlock/AsideBlock.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+$asideblock-default-height: calc(100% - 62px);
+$asideblock-default-width: 280px;
+$asideblock-max-width: 70%;
+$asideblock-default-box-shadow: 0 12px 12px 6px rgba(0, 0, 0, 0.08);
+$asideblock-menu-content-z-index: 100;
```

Comment:
All z-indexes should be moved to `src/styles/_variables.scss` under `/* z-index */` section. Easier to handle them then when app grows.